docs/collecting-data/collecting-from-own-applications/mobile-trackers/previous-versions/mobile-trackers-v2-x/tracking-events/index.md

```diff
@@ -286,7 +286,7 @@ Each custom context is an array of self-describing JSON following the same patte
 
 Here are two example custom context JSONs. One describes a screen:
 
-```
+```javascript
```

Comment:
Why not json?

Reply:
So, these examples are not valid json, as they have no quotes. They are valid syntax in many languages though. Due to this, if you make them `json` then they don't render as you'd expect.

`javascript`
<img width="957" alt="Screenshot 2022-09-14 at 22 09 23" src="https://user-images.githubusercontent.com/8820009/190263328-f84b818b-cf2d-4db7-951c-6f01caefa648.png">

`json`
<img width="970" alt="Screenshot 2022-09-14 at 22 09 10" src="https://user-images.githubusercontent.com/8820009/190263349-87c35a24-0926-450e-ad14-7074564b043e.png">

Fixed `json`
<img width="963" alt="Screenshot 2022-09-14 at 22 10 08" src="https://user-images.githubusercontent.com/8820009/190263373-5e360dfd-d4ac-4237-9eba-2be9d8a01333.png">

However, to do the latter, I need to add quotes which now means it potentially doesn't copy and paste nicely into a programming language. It's probably a bit pointless here, because it's in the mobile section and you ain't copy and pasting this anywhere in Obj-C. 

So thats why I picked Javascript. I could add _lots_ of quotes to all the examples but picking `javascript` seemed easier.